platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLafPrefs.java

```diff
@@ -35,6 +37,28 @@ class FlatLafPrefs {
 
     private static final Preferences prefs = NbPreferences.forModule(FlatLafPrefs.class);
 
+	static Color getAccentColor() {
+        return parseColor(prefs.get(ACCENT_COLOR, null));
```

Comment:
What happens in this case when the "default" color was selected? Does the accent color then end up being taken from the various FlatLAF.properties files, or does it end up being explicitly set to the zero-alpha DEFAULT color, with invisible accents?